### ISSUE TITLE
Feature: An Uninstall Prompt

### ISSUE THREAD

**Issue by Alipoodle** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1471]
This may have been a problem back in 3.4.0.

I had installed 3.4.0 on my computer and had left for a while to go on a trip, once i had got back I couldn't use GPMDP As once it loaded onto the main screen it would constantly load. I checked on the here to see if there's a new release (3.4.3)
Although when i tried to uninstall the 3.4.0 i wasn't given any Uninstall Prompt of any kind. Not even "Are you sure you want to uninstall?" or Even an Administrator Prompt (Which i'm aware you want to keep GPMDP from needing)

**Comment by MarshallOfSound** [comment]
I think this falls in line with my philosophy of "Let's trust that the user kinda knows what they are doing".

In order to uninstall GPMDP you need to specifically open Control Panel and click uninstall (there is no other way).  No accidental shortcut to click, nothing.

TLDR: There isn't actually a way to do this with Squirrel, and even if there was, let's trust that the user knows vaguely what they are doing.

In regards to no update, in the future you can install a new version over the top of the old one (no uninstall required) and are you behind a VPN or proxy?

**Comment by Alipoodle** [comment]
No VPN / Proxy

**Comment by MarshallOfSound** [comment]
@Alipoodle How long did you wait for the update notification.  It can take up to 10 minutes to download, extract and prepare the update.  Depending on system speed and network speed.

**Comment by Alipoodle** [comment]
As i normally open GPMDP From booting i would say that it would of been more than 10 minutes.
And i'm on 200mbps~ connection (Direct connection)
